### PR TITLE
add Thales token to the token list

### PR DESCRIPTION
**Pull request by Red-Thales**
**Description**
A clear and concise description of the features you're adding in this pull request.
Adding the Thales token for the Thales Protocol
**Additional context**
Add any other context about the problem you're solving.

**Metadata**
- Fixes #[Link to Issue]

### CODE REVIEW

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -1199,11 +1199,33 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x03e173ad8d1581a4802d3b532ace27a62c5b81dc",
+      "name": "Thales",
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:1206]: Set the name to match L1: `Thales Token`

File: optimism.tokenlist.json [hunk 2]
```diff
@@ -1199,11 +1199,33 @@
...
+      "symbol": "THALES",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/THALES.svg",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0x217D47011b23BB961eB6D93cA9945B7501a5BB11",
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:1216]: This token is pointing to a different L1 Thales token at `0x8947da500eb47f82df21143d0c01a29862a8c3c5` not the `0x03e173ad8d1581a4802d3b532ace27a62c5b81dc` address above. Which is the correct L1 token you want to bridge? And is there a reason why you deployed a dedicated Optimism Thales token on L1?

Comment (elenadimitrova) [reply optimism.tokenlist.json:1216]: Also note that you can use the token factory described [here](https://community.optimism.io/docs/guides/token-dev/#getting-some-testnet-eth) to create your L2 token.

File: logos/THALES.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...62 7.633c0 5.962 4.833 10.795 10.794 10.795 2.981 0 5.68-1.208 7.633-3.162l-.569-.569zM220.845 44.816H200.87v-34.16a4.5 4.5 0 0 0-9 0v38.66a4.5 4.5 0 0 0 4.5 4.5h24.476a4.5 4.5 0 1 0-.001-9zM260.087 6.156h-25.135a4.5 4.5 0 0 0 0 9h25.135a4.5 4.5 0 0 0 0-9zM254.171 34.486a4.5 4.5 0 0 0 0-9h-13.305a4.5 4.5 0 0 0 0 9h13.305zM260.087 44.816h-25.135a4.5 4.5 0 0 0 0 9h25.135a4.5 4.5 0 0 0 0-9zM135.77 6.156a4.5 4.5 0 0 0-4.5 4.5v14.708h-19.208V10.656a4.5 4.5 0 0 0-4.5-4.5H66.904a4.5 4.5 0 0 0 0 9h12.671v34.16a4.5 4.5 0 0 0 9 0v-34.16h14.487v34.16a4.5 4.5 0 0 0 9 0V34.364h19.208v14.953a4.5 4.5 0 0 0 9 0v-38.66a4.5 4.5 0 0 0-4.5-4.501zM298.084 25.486H282.12a2.997 2.997 0 0 1-2.994-2.994V18.15a2.997 2.997 0 0 1 2.994-2.994h16.64a4.5 4.5 0 0 0 0-9h-16.64c-6.613 0-11.994 5.38-11.994 11.994v4.343c0 6.613 5.381 11.994 11.994 11.994h15.964a2.997 2.997 0 0 1 2.993 2.994v4.343a2.997 2.997 0 0 1-2.993 2.994h-23.458a4.5 4.5 0 0 0 0 9h23.458c6.613 0 11.993-5.38 11.993-11.994V37.48c0-6.613-5.38-11.994-11.993-11.994zM169.723 8.898a4.5 4.5 0 0 0-8.284 0l-16.41 38.66a4.499 4.499 0 0 0 4.142 6.257h32.82a4.5 4.5 0 0 0 4.142-6.257l-16.41-38.66zm-13.755 35.919 9.613-22.647 9.613 22.647h-19.226z"></path></svg>
```
Comment (elenadimitrova) [inline logos/THALES.svg:1]: Your image should be a square, I think this won't display properly

**Review by elenadimitrova — CHANGES_REQUESTED** [review]

File: optimism.tokenlist.json (hunk 1 above)
Comment (Red-Thales) [reply optimism.tokenlist.json:1206]: Changed it to `Optimistic Thales Token` to match the L1 contract. Also posted the same change with the L2 token.

File: optimism.tokenlist.json [hunk 4]
```diff
@@ -1199,11 +1199,33 @@
...
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/THALES.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0x217D47011b23BB961eB6D93cA9945B7501a5BB11",
+      "name": "Optimistic Thales Thales",
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:1217]: Please check the name -> should be "Optimistic Thales Token"

**Review by elenadimitrova — CHANGES_REQUESTED** [review]
LGTM except for that token name on L2. Also the image might not display properly with that size, pls shrink to 256x256 or thereabouts.

File: optimism.tokenlist.json (hunk 4 above)
Comment (Red-Thales) [reply optimism.tokenlist.json]: Should be all good to go now, thanks a ton for the help & patience Elena. This PR was a first for me, hopefully with more to come.

**Review by elenadimitrova — APPROVED** [review]

**Comment by Red-Thales** [comment]
@elenadimitrova 

On the gateway page the L2 token doesn't have the same name than the ones in our contracts & PR, I believe this is causing the gateway page to crash both on L1 and L2 when selecting THALES. That said, everything looks good on with optimism tokenlist on Uniswap.

**Comment by elenadimitrova** [comment]
@Red-Thales indeed the name mismatch was causing a problem. I have set them here https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/74 to be the same, let me know if you still have problems or if you want to call the token something else.

**Pull request merged**